src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
Also, on a platform that is unknown to us, we cannot guarantee even the last known "lowest RID" specific assets are guaranteed to work.